example-cropping/README.md

```diff
@@ -0,0 +1,779 @@
+  async function signUp(e) {
+    e.preventDefault()
+    if (loading) return;
+
+
+    setError(false)
+    setLoading(true)
+
+    if (!(password.length >= 6 && password.length <= 32)) {
+
+      setError('Error: Password must be between 6 and 32 characters.')
+      setLoading(true)
+
+      return;
+    }
+    try {
+
+      await props.appwrite.account.create(
+        email,
+        password
+      );
+      props.setCurrentPage(props.currentPage)
+
+    } catch (err) {
+      setError(err.message)
+
+    }
+    setLoading(false)
+  }
+
+
+
+
+  return (
+    <Container component="main" maxWidth="xs"  style={{ display: props.currentPage ? "none" : "block" }}>
+    <CssBaseline />
+    <div className={classes.paper}>
+      <Typography component="h1" variant="h5">
+        Sign Up
+      </Typography>
+      
+      {error && (
+           <Alert severity="error">{error}</Alert>
+    )}
+      <form className={classes.form} noValidate onSubmit={(e) => signUp(e)}>
+        <TextField
+        onChange={(event) => setEmail(event.target.value)} 
+          variant="outlined"
+          margin="normal"
+          required
+          fullWidth
+          id="email"
+          label="Email Address"
+          name="email"
+          autoComplete="email"
+          autoFocus
+        />
+        <TextField
+        onChange={(event) => setPassword(event.target.value)}
+          variant="outlined"
+          margin="normal"
+          required
+          fullWidth
+          name="password"
+          label="Password"
+          type="password"
+          id="password"
+          autoComplete="current-password"
+        />
+       
+        <Button
+          type="submit"
+          fullWidth
+          variant="contained"
+          color="primary"
+          className={classes.submit}
+          disabled={loading}
+        >
+          Sign Up
+        </Button>
+        <Grid container>
+          
+          <Grid item>
+            <Link href="#" variant="body2" onClick={()=>props.setCurrentPage(props.currentPage)}>
+              {"have an account? Login"}
+            </Link>
+          </Grid>
+        </Grid>
+      </form>
+    </div>
+  </Container>
+  )
+
+};
+
+export { SignUp };
+```
+example usage
+
+```js
+<SignUp
+            appwrite={appwrite}
+            currentPage={currentPage}
+            setCurrentPage={(currentPage) => setCurrentPage(!currentPage)}
+          />
+```
+it takes three props which is 
+
+- ```appwrite``` to connect to appwrite server using sdk
+- ```currentPage``` the state of login or sign up
+- ```setCurrentPage``` to switch between login and sign up
+
+so what this does is process the sign up through ```signUp``` function then if success it will go to login component so that we can login using that credential notice that we dont have email verification here because the purpose of the this tutorial is actually to cropping an image and this step is one of the step to implement that functionality.
```

Comment:
```suggestion
So what this does is process the sign up through ```signUp``` function then if success it will go to login component so that we can login using that credential notice that we dont have email verification here because the purpose of the this tutorial is actually to cropping an image and this step is one of the step to implement that functionality.
```